zeppelin-zengine/src/test/java/org/apache/zeppelin/storage/LocalConfigStorageTest.java

```diff
@@ -0,0 +1,50 @@
+package org.apache.zeppelin.storage;
+
+import org.apache.commons.io.IOUtils;
+import org.junit.Test;
+
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.IOException;
+import java.io.InputStream;
+import java.nio.file.Files;
+import java.nio.file.Path;
+
+import static org.junit.Assert.*;
+
+public class LocalConfigStorageTest {
+    public static final String TEST_STRING = "this is a test!";
+
+    @Test
+    public void testWritingAtomically() throws IOException {
+        final Path destination = Files.createTempFile("test-", "file");
+        final File destinationFile = destination.toFile();
+        try {
+            LocalConfigStorage.atomicWriteToFile(TEST_STRING, destinationFile);
```

Comment:
I think it would be better to make those methods of `atomicWriteToFile` and `readFromFile` as a package local. `LocalConfigStorage` have `ZeppelinConfiguration` only and it could be initiated easily.

Reply:
@jongyoul Sorry, you mean to move them into a separate class that has package local visibility?

Reply:
@jongyoul these functions are already have package visibility...